src/storer.ts

```diff
@@ -0,0 +1,53 @@
+import {WebPage} from "./web-page";
+export class Storer {
+
+    static azure = require('azure');
+    static crypto = require('crypto');
+    static config = require('../config.json');
+
+    static blobService = Storer.azure.createBlobService(Storer.config.storageAccountname,
+        Storer.config.storageKey);
+
+    static container = Storer.config.container;
+
+    public static storeWebsite(webpage : WebPage) : void {
+        Storer.storeWebsiteBlob(webpage);
+
+        //TODO: Create database entry for website
+    }
+
+    /**
+     * Stores given website as a blob in the azure blob storage. The filename will be an md5 hash
+     * of website uri + content
+     * @param webpage       WebPage object to be stored
+     * @param callback      Optional callback param that will receive the filename as a parameter in case of success
+     */
+    private static storeWebsiteBlob(webpage : WebPage, callback? : (err? : Error, blobName? : string) => void) : void {
+        let blobName = Storer.hashWebsite(webpage);
+        let blobContent = '';
+        for (let property in webpage.headers) {
+            if (webpage.headers.hasOwnProperty(property)) {
+                blobContent += property + ": " + webpage.headers[property] + '\n';
+            }
+        }
+        blobContent += webpage.content
+        console.log('blobName: ' + blobName);
+        console.log('blobContent: ' + blobContent);
```

Comment:
Can you remove the logging? Since we've got a callback now, that shouldn't be necessary as it can be passed as the callback if needed. I am also planing to get rid of all the logging and reserve the console for logging unhandled errors only.